src/features/dashboard/components/AppsTable/index.tsx

```diff
@@ -276,6 +338,15 @@ const AppsTable = ({ apps }: AppsTableProps) => {
     };
   };
 
+  const getAppsTableOptionProps = () => {
```

Comment:
@ahmed-deriv can we use useCallback here as well to stop unwanted function calls on re-render?

Reply:
Fixed